app/src/androidTest/java/com/example/diceroller/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,25 @@
+package com.example.diceroller;
+
+import android.content.Context;
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
```

Comment:
Are these tests the ones in the codelab?

Reply:
Yes, these tests are referenced/viewed in the next pathway. I'm not sure why the initial commit didn't have them, but they needed to be added.